examples/13. Named Endpoints.md

```diff
@@ -6,6 +6,7 @@ This API example demonstrates how to define a standalone endpoint with an identi
 ## API Blueprint
 + [Previous: Advanced Action](12.%20Advanced%20Action.md)
 + [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/13.%20Named%20Endpoints.md)
++ [Next: JSON Schema](14.%20JSON%20Schema.md)
```

Comment:
@pksunkara I think in this case it is OK if this goes last (as an advanced topic).